services/121-service/src/programs/programs.controller.ts

```diff
@@ -56,6 +56,7 @@ export class ProgramController {
     private readonly koboConnectService: KoboConnectService,
   ) {}
 
+  // XXX: I'm not sure whether this needs to be changed and if so, how
   // Note: protecting this endpoint because we assume in this branch the PA-app will be removed
```

Comment:
See comment above

Reply:
No idea, I couldn't help 😞

Reply:
You can leave it like this. We had a discussion about whether we should add an extra read program permission, but it seems so trivial we decided against it. (there is no secret data in the program and every roles should have that permission to be able to use the platform). Maybe just remove your comment @aberonni